### ISSUE TITLE
bug: fails to initialize because keystore can not be loaded from jar location

### ISSUE THREAD

**Issue by skuzzle** [skuzzle/spring-boot-wiremock#2]
The problem is this:
```java
        final Resource resource = resourceLoader.getResource(location);
        try {
            return resource.getFile().toString();
        } catch (final IOException e) {
            throw new IllegalArgumentException("Error resolving location for property " + key, e);
        }
```
`getFile()` will throw an exception when loading from jar

ALso hard to detect in local tests because these file are always available from filesystem in `src/main/resources`

**Comment by skuzzle** [comment]
Fixed in 0.0.14

**Comment by skuzzle** [comment]
Fixed in `0.0.16`
